CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 - New #23: Add `DbSchemaManager` and SQL dumps for working with schema (@arogachev)
 - Chg #23: Remove CLI dependencies (@arogachev)
+- Enh #60: Decouple storages (@arogachev)
```

Comment:
```suggestion
- Enh #60: Make database tables independent (remove foreign keys) (@arogachev)
- New #60: Add `TransactionalManagerDecorator` (@arogachev)
```

Reply:
It's not about database tables and foreign keys only.

Reply:
Please, add what I missed.